examples/external_ros_resources/drake_example_pendulum/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>drake_example_pendulum</name>
+  <version>0.1.0</version>
+  <description>Example showing usage of resource from an external ROS package in Drake</description>
+  <author email="drake-users@mit.edu">Drake Users</author>
+  <maintainer email="drake-users@mit.edu">Drake Users</maintainer>
+  <license>Apache License 2.0</license>
+  <url type="website">https://drake.mit.edu</url>
+  <url type="bugtracker">https://github.com/RobotLocomotion/drake-ros/issues</url>
+  <url type="repository">https://github.com/RobotLocomotion/drake-ros</url>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <exec_depend>drake_example_pendulum_description</exec_depend>
```

Comment:
Mind adding `<depend>drake</depend>`? With that I'm able to build this together with `RobotLocomotion/drake` in an isolated colcon workspace.